engines/wintermute/platform_osystem.cpp

```diff
@@ -144,25 +144,34 @@ void BasePlatform::handleEvent(Common::Event *event) {
 // Win32 API bindings
 //////////////////////////////////////////////////////////////////////////
 bool BasePlatform::getCursorPos(Point32 *lpPoint) {
+#ifndef ENABLE_WME3D
 	BaseRenderOSystem *renderer = static_cast<BaseRenderOSystem *>(_gameRef->_renderer);
 
+#endif
 	Common::Point p = g_system->getEventManager()->getMousePos();
 	lpPoint->x = p.x;
 	lpPoint->y = p.y;
 
+#ifndef ENABLE_WME3D
```

Comment:
It isn't immediately obvious why this is ifndef-ed out. Perhaps add a comment to explain why.